src/kajol/FindPrimeNumbersBetweenUserDefinedRange.java

```diff
@@ -0,0 +1,34 @@
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100*/
+
+package kajol;
+
+import java.util.Scanner;
+
+public class FindPrimeNumbersBetweenUserDefinedRange {
+	
+	void findPrimeNumbers(int start,int end) {
+		System.out.println("Prime Numbers Between "+start+" and "+end+" :");
+		for(int index=start;index<=end;index++) {
+			boolean flag=true;
+			for(int innerIndex=2;innerIndex<=Math.sqrt(index);innerIndex++) {
+				if(index%innerIndex==0)
+					flag=false;
+			}
```

Comment:
Code improvement part: You should put break here, because once you know this number is not prime number then you can break no require to check with further numbers